docker/oso-rhel7-zagg-client/vendor/prometheus_client/__init__.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
```

Comment:
Consider adding a readme file to this directory saying where this library came from (URL), to make it easier for us to know where to go to update it.